src/APIFunctions/AWSRekognition.js

```diff
@@ -0,0 +1,197 @@
+/* eslint-disable */
```

Comment:
get rid of eslint-disable for the entire file. it's okay to use it while you're developing. consider renaming the file so it doesn't include `AWS`